### DISCUSSION TITLE
Consider inverting the alt text indicator

### DISCUSSION THREAD

**Discussion by evant** [tuskyapp/Tusky#3080] [Ideas]
For the recently added image alt text indicator, I'm wondering if it would be more useful to show something for images without alt text instead of ones that do. In an ideal world every image would have alt text which would make an indicator redundant, so it would act more as a nudge towards that direction.

**Comment by Lakoja** [comment]
(Just as additional information)

There is kind of this nudge when composing a status with image: The top line will show a warning icon.

However that is probably way too "hidden" to have any effect.

The actual ALT tag I intended as an inverse nudge: "What do the other images have that mine doesn't?"
